api/sys_plugins.go

```diff
@@ -16,34 +18,21 @@ type ListPluginsInput struct {
 
 // ListPluginsResponse is the response from the ListPlugins call.
 type ListPluginsResponse struct {
-	// Names is the list of names of the plugins.
-	NamesByType map[consts.PluginType][]string `json:"names"`
-}
-
-type listResult struct {
-	Data struct {
-		Keys []string `json:"keys"`
-	} `json:"data"`
-}
-
-type readResult struct {
-	Data map[string]interface{} `json:"data"`
+	// PluginsByType is the list of plugins by type.
+	PluginsByType map[consts.PluginType][]string `json:"types"`
```

Comment:
I changed the JSON value from `names` to `types`, since we are already breaking the API by changing the struct type returned. If we want to preserve `names`, I can revert the change.

Reply:
I think that's an equally good choice, so why don't we stick with types?